exporter/datadogexporter/internal/metrics/consumer.go

```diff
@@ -108,6 +108,22 @@ func (c *Consumer) ConsumeTimeSeries(
 			Type: datadog.PtrString("host"),
 		},
 	})
+
+	// datadog-api-client-go does not support `origin_product`, `origin_sub_product` or `origin_product_detail`.
+	// We add them as 'AdditionalProperties'. This is undocumented; it adds the fields to the JSON output as-is:
+	// https://github.com/DataDog/datadog-api-client-go/blob/f692d3/api/datadogV2/model_metric_origin.go#L153-L155
```

Comment:
Should it be documented somewhere? I'm wondering how likely it is that this comment will be lost in the future, or go stale.

Reply:
I can report this to the datadog-api-client-go maintainers, good idea, I will link it here :)